nucypher/blockchain/eth/policies.py

```diff
@@ -127,7 +127,7 @@ def __init__(self,
         self.handpicked_ursulas = handpicked_ursulas or UNKNOWN_ARRANGEMENTS
         self.value = value
         self.author = alice
-        self.selection_buffer = 1.5
+        self.selection_buffer = 1
```

Comment:
So we no longer want to sample for extra ursulas just in case i.e. for `n` ursulas we only look for `n` and no backups - was this causing an unintended issue?

Reply:
Yes, it's a temporary workaround while we have such a small number of nodes in testnet. Once we have, say, >10 nodes, then we can restore this.

Reply:
Should we use an envvar or a flag (or something else) so that we don't mistakenly forget 😄 ?

Reply:
There's this open issue https://github.com/nucypher/nucypher/issues/1090

Reply:
Ah - 👍

Reply:
That being said, the probability of not getting 10 unique nodes with this will be really, really high with this (so, quite many failures)

Reply:
This is a duplicated change in #1225.